test/regressions/index.test.ts

```diff
@@ -119,6 +121,11 @@ async function main() {
             return;
           }
 
+          if (codesandboxIframeDemos.some((demo) => route.includes(demo))) {
+            // Ignore codesandbox embedded demos since they're not using packages built from the branch anyway.
+            return;
```

Comment:
I'm surprised to see all these blacklist patterns here. I thought we completely switched to negative globs in https://github.com/mui/mui-x/blob/master/test/regressions/testsBySuite.ts. Those are much better as they just avoid bundling the unused demos altogether, which is better for performance.

Reply:
Updated 👍🏻